README.md

```diff
@@ -1,72 +1,76 @@
-# hyperbahn
+# Hyperbahn
 
-Service discovery and routing solution.
+Service discovery and routing system for large-scale microservice operations.
+* Unanswered questions? Open a new [Github issue][issues].
+* Hyperbahn will be on [Uber's Open Source site][oss] soon!
 
 ## Overview
 
-Hyperbahn is a service discovery and routing network. Its aim is to simplify
-reliably talking to any service without having to know where the service is
-running.
+Hyperbahn enables service discovery and routing for large-scale systems
+comprised of many microservices. Distributed, fault tolerant, and highly
+available, it lets one service find and communicate with others simply and
+reliably without having to know where those services run.
 
 Hyperbahn is an overlay network of routers designed to support the
-[TChannel RPC protocol][tchannel]. Hyperbahn itself is a [ringpop][ringpop]
-based mesh network of router nodes that dynamically converge and gossip known
-services onto a consistent hash ring topology.
+[TChannel RPC protocol][tchannel]. Hyperbahn itself is based on
+[Ringpop][ringpop]. Its router nodes dynamically converge and gossip
+known services onto a consistent hash ring topology, forming a
+mesh of services that can send requests to each other without human
+intervention or knowledge of specific ports and addresses.
 
 Hyperbahn and TChannel clients are currently supported
 in Go, Python, and Node.js.
 
 ## Hyperbahn Features
 
-Hyperbahn provides many scaling and fault tolerance features including:
+Whether you're making your service available to others, or you're a service
+consumer, Hyperbahn comes with many features to make application development easier:
 
- - timeouts to transitively enforce your service's SLA
- - retries to eliminate transient failures within a request timeout window
- - load balancing so that calls are spread out evenly across your service
- - rate limiting to shield your service from excessive calls
- - circuit breaking to prevent cascading failure
- - circuit breaking to cut off broken clients
+ - **Configuration Discovery** - Stop managing host/port files or configurations.
+ - **Timeouts** - Transitively enforce services' SLAs. Every request
+   must specify a timeout, moving towards a fast failure model.
+ - **Retries** - Eliminate transient failures within a request timeout window.
+ - **Load Balancing** - Spread calls evenly across a service, and distribute outgoing
+   requests evenly over connected Hyperbahn nodes.
+ - **Rate Limiting** - Shield services from excessive calls.
+ - **Circuit Breaking** - Prevent cascading failure and cut off broken clients. Build
+   fast failure into all layers of the network.
 
```

Comment:
I would also add:

**Distributed Tracing** - Understand the entire call flow using Zipkin.